installers/mac-arm/build-ballerina-macos-arm-x64.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+function printUsage() {
```

Comment:
Can't we use the same script and generate 2 installers by switching zip source files? 
We may have to parameterize few things to support both